support-frontend/tsconfig.json

```diff
@@ -15,6 +15,7 @@
 		"paths": {
 			"ophan": ["../node_modules/ophan-tracker-js/build/ophan.support"],
 			"@modules/product-catalog/*": ["../node_modules/@guardian/support-service-lambdas/modules/product-catalog/src/*"],
+			"@modules/internationalisation/*": ["../node_modules/@guardian/support-service-lambdas/modules/internationalisation/src/*"],
```

Comment:
This was necessary as the code we import from the support-service-lambdas package for the product catalog now itself imports `@modules/internationalisation` code, without this TS fails to find the relevant code.